tests/pytorch/test_sanity.py

```diff
@@ -989,3 +992,102 @@ def test_sanity_fp8_gemm_with_unalignment(N, datatype):
             use_bias=False,
             use_split_accumulator=False)
     torch.cuda.synchronize()
+
+@pytest.mark.skipif(not fp8_available, reason=reason_for_no_fp8)
+@pytest.mark.skipif(get_device_compute_capability() != (9, 0), reason="FP8 tests require Hopper.")
```

Comment:
```suggestion
@pytest.mark.skipif(not fp8_available, reason=reason_for_no_fp8)
```

Reply:
Is there reason we were specifically exclude ada for this test?

Reply:
Our kernels only work for sm90 as I just checked with cuDNN.